lib/minio/client.rb

```diff
@@ -147,8 +147,14 @@ def list_objects(bucket_name, folder_path, max_keys: 1000)
     objects
   end
 
-  def send_request(method, uri, body = nil)
-    headers = @signer.build_headers(method, uri, body, @creds, REGION)
+  def set_lifecycle_policy(bucket_name, policy_id, expiration_days)
+    policy = "<LifecycleConfiguration><Rule><ID>#{policy_id}</ID><Status>Enabled</Status><Filter></Filter><Expiration><Days>#{expiration_days}</Days></Expiration></Rule></LifecycleConfiguration>"
+    response = send_request("PUT", s3_uri("#{bucket_name}?lifecycle"), policy, true)
+    response.status
+  end
+
+  def send_request(method, uri, body = nil, needs_md5 = false)
```

Comment:
Using unnamed positional boolean parameters can make method usage somewhat difficult. Should we use a named parameter for `needs_md5`?

```suggestion
  def send_request(method, uri, body = nil, needs_md5: false)
```

Reply:
I also prefer named parameters, but it is not used in this file at all. I tried to follow local convention in this file. @furkansahin was that intentional? Would you be OK if we switch to named parameters here?

Reply:
Yeah, it's alright, we can create a refactoring PR later on to change the convention in the file as well. This is safer, let's switch.